### ISSUE TITLE
Sourcemaps for the yaml server project

### ISSUE THREAD

**Issue by brifly** [redhat-developer/vscode-yaml#745]
## Summary
I'd like to contribute to this project but at the moment Im having some issues with getting sourcemaps working with the `yaml-language-server` project.

I have gone through the steps on the contributing.md and I can get the extension to run in the debugger fine. I can step through the extension ok and set breakpoints `vscode-yaml`. Breakpoints fail to bind on `yaml-language-server` however. I can make changes and see `console.trace()` output in the debug console however and if I force it to break using a debugger statement it will break in `vscode-yaml\dist\languageserver.js`. The dist folder in the `vscode-yaml` does contain a sourcemap for `languageserver.js`. The `yaml-language-server` `out/server/src`

Is there anything else I need to do to be able to debug the language server ts files when running through the vscode-yaml plugin?

## Relevant information
I had an earlier problem with the default port that inspect listening on so I changed it in both `launch.json` of the server project and in `/src/node/yamlClientMain.ts`.

I've tried both windows and wsl, both have the same issue.

**Comment by gorkem** [comment]
@evidolob Can you take a look at this when you have time? I remember seeing this behaviour after switching to webpacks.

**Comment by evidolob** [comment]
@gorkem That was intentionally, as we want to reduce the size of extension bundle, so we not include source maps.

**Comment by brifly** [comment]
I managed to resolve this. The problem is the `outFiles` in `launch.json` of the `yaml-language-server` project.
It should be:
```
{
      "name": "Attach to server",
      ...
      "outFiles": ["${workspaceRoot}/../vscode-yaml/dist/*.js"]
      ...
}
```

It currently points to "${workspaceRoot}/out/server/**/*.ts" from my reading of the docs (https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_launch-configuration-attributes) this looks incorrect. OutFiles should point to the generated .js files i.e. in this case the dist folder of the vscode-yaml project. 

I also found its a bit easier to set `config.entry.languageserver` in the webpack config of `vscode-yaml` to `./../yaml-language-server/src/server.ts`. This way the language server gets built along with the plugin and every change only requires one build.
